tests/10-issuer.js

```diff
@@ -167,14 +168,12 @@ describe('Issue Credential - Data Integrity', function() {
           shouldThrowInvalidInput({result, error});
         }
       });
-      it('credential MAY have property "issuanceDate"', async function() {
+      it.skip('credential MAY have property "issuanceDate"', async function() {
```

Comment:
Why is this being skipped?

Reply:
it turns out that while `issuanceDate` is not list as required in the `VC-API` spec, is required by the `vc-data-model` spec. So we're now adding `issuanceDate` all of the time to the tests. I guess this could be deleted, but I thought it was safer to skip for now.

Reply:
Ah okay, please do add a comment why its being skipped then.

Reply:
comment added here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/24/commits/dd5b9fffcd8c89c3c0ab13fa75c86f2bb20d8345